msal4j-sdk/pom.xml

```diff
@@ -58,7 +59,7 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <version>1.18.36</version>
-            <scope>provided</scope>
+            <scope>test</scope>
```

Comment:
When can this be removed?

Reply:
There will likely be one final PR removing all the leftover dependency references, I just wanted to keep the number of changed files in each PR as low as possible for easier review and avoid duplicate changes/complicated merging

Some of the infrastructure used by tests to get tokens/users/etc. use lombok and other dependencies, but it doesn't really affect the tests themselves